extension/src/experiments/model/index.ts

```diff
@@ -857,22 +857,34 @@ export class ExperimentsModel extends ModelWithPersistence {
     return commitsBySha
   }
 
+  private addBranchToFlattenedCommit(commitAndExps: Commit[], branch: string) {
+    return commitAndExps.map(commitOrExp => ({
+      ...commitOrExp,
+      branch: commitOrExp.branch || branch,
+      otherBranches: commitOrExp?.otherBranches
```

Comment:
[I] At first glance a much simpler data structure would be `flatBranches?: string[]`. Don't add a branch to these records at all.

Reply:
`flatBranches` is also a breadcrumb as to why we need the variable.

Reply:
Updated and definitely simplified the code on the backend and frontend!